README.adoc

```diff
@@ -0,0 +1,90 @@
+= IoT OpenShift Demo
+===============
+
+This demo showcases Intelligent IoT Gateway on the Red Hat OpenShift Container Platform. The Intelligent IoT Gateway enables real time decision-making at the edge. OpenShift provides secure enterprise-grade container platform to accelerate IoT application delivery. Through the use of containers and microservices, this demo enables security, scalability and ease of deployment.
+
+image::images/IoT-OSE-Containers.png[width="462", height="614", align=center]
+
+*Please note: This repository is currently under development and the feature set/components are subject to change without notice*
+
+== Components
+
+* https://access.redhat.com/documentation/en/red-hat-xpaas/version-0/red-hat-xpaas-fuse-integration-services-image/[Fuse Integration Services]*
+Red Hat JBoss Fuse Integration Services provides a set of tools and containerized xPaaS images that enable development, deployment, and management of integration microservices within OpenShift.
+* https://access.redhat.com/documentation/en/red-hat-xpaas/0/paged/red-hat-xpaas-a-mq-image/[AMQ and MQTT]
+* https://access.redhat.com/documentation/en/red-hat-xpaas/0/paged/red-hat-xpaas-a-mq-image/[Realtime Decision Server]
+* https://zeppelin.apache.org/[Apache Zeppelin]
+* Simulated Software Sensor - Need a description
+
+== Prerequisites
+
+* Git
+* Access to an OpenShift environment
+
+_Note: The OpenShift environment can be enabled by CDK platform. To learn how to install CDK, check out https://github.com/redhatdemocentral/cdk-install-demo[CDK-Install Demo]_
+
+* OpenShift Command Line Interface (CLI) - Do we still need this?
+
+== Setup
+*Step 1:* Launch the OpenShift Container platform and login to OpenShift Command Line
+
+*Step 2:* Download the iot-ose project in the OpenShift instance:
+
+ [user@localhost ~]$ git clone https://github.com/sabre1041/iot-ose.git
+
+
+The https://github.com/sabre1041/iot-ose/blob/master/init.sh[init.sh] script, provided with the project, automates the provisioning process for the demo.
+
+*Step 3:* Execute the init script
+
+ [user@localhost ~]$ ./init.sh
+ 
+The init script triggers asynchronous builds and deployments of A-MQ, Decision Server, Fuse Integration Service, PostgreSQL, Zeppelin and Software Sensor components. OpenShift web console provides a graphical UI into the setup.
+
+*Step 4:* Login to the OpenShift web console using a web browser at 10.1.2.2:5443 (login/password - admin/admin) and select the **iot-ose** project
```

Comment:
This assumes that you are using the CDK. I would genericize to be "Launch the OpenShift web console and login..."